spec/controllers/admin/companies_controller_spec.rb

```diff
@@ -32,7 +32,17 @@
 
   describe "PATCH #update" do
     it "updates the status of a company" do
-      patch :update, id: company.id, company: { status: "Contacted" }
+      patch :update, id: company.id, company: { status: "contacted" }
```

Comment:
Changed from Caps "Contacted" to "contacted" because that's what is included in validations and won't take if Cap "C" is provided